vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -684,12 +684,21 @@ func (s *Scheme) PrioritizedVersionsForGroup(group string) []schema.GroupVersion
 // PrioritizedVersionsAllGroups returns all known versions in their priority order.  Groups are random, but
 // versions for a single group are prioritized
 func (s *Scheme) PrioritizedVersionsAllGroups() []schema.GroupVersion {
+	hasLegacyV1 := false
 	ret := []schema.GroupVersion{}
 	for group, versions := range s.versionPriority {
 		for _, version := range versions {
+			if len(group) == 0 && version == "v1" {
```

Comment:
What's the use case? None of the groups should be more important than others, right?

Reply:
> What's the use case? None of the groups should be more important than others, right?

corev1.List is actually a metainternal.List.  That list is registered as "unversioned", which means it serializes using the first group it sees.  Until we fix our construction of corev1.Lists and printers in oc, we need something to help us serialize to the correct version.